CHANGES

```diff
@@ -4,6 +4,11 @@
 
  Changes between 1.0.2g and 1.1.0  [xx XXX xxxx]
 
+  *) --strict-warnings no longer enables runtime debugging options
+     such as REF_DEBUG. Instead, REF_DEBUG is enabled automatically
```

Comment:
Me either. :(  We should turn on *_DEBUG if --debug is built.  But a quick grep shows that the naming of DEBUG ifdef's is not consistent (ENGINE_REF_COUNT_DEBUG, OPENSSL_DEBUG_PKCS5V2, 
OPENSSL_(INIT|POLICY)_DEBUG, etc.  Ugh.  Separate MR and discussion about this.